README.md

```diff
@@ -8,6 +8,8 @@ synchronizes data from your UI (view) with your JavaScript objects (model) throu
 binding. To help you structure your application better and make it easy to test, AngularJS teaches
 the browser how to do dependency injection and inversion of control.
 
+HTML is also used to determine the execution of the app. Special attributes in the HTML determine which controllers to use for each element. These attributes determine "what" gets loaded, but not "how". This declarative approach greatly simplifies app development in a sort of WYSIWYG way. Rather than spending time on how the program flows and what should get loaded first, you simply define what you want and Angular will take care of the dependencies.
```

Comment:
I feel this goes into a bit too much detail for this high-level description here.
Maybe incorporate it into the `Interconnection with HTML at the root level` section below (or in it's own section).